packages/@coorpacademy-components/src/organism/accordion/toggler/index.js

```diff
@@ -32,16 +32,20 @@ class AccordionToggler extends React.Component {
   }
 
   render() {
-    const {children = []} = this.props;
+    const {children = [], moreIconType, lessIconType} = this.props;
 
     return (
-      <Accordion tabProps={this.state.tabProps} onClick={this.handleOnClick}>
+      <Accordion
+        moreIconType={moreIconType}
```

Comment:
I don't want to be able to change the icons

Reply:
I created a new component, which is accordion-cohort

Reply:
rollback default